changelog.md

```diff
@@ -1,13 +1,22 @@
 # Change Log
 
 ## Unreleased
-* Fix: Internal warnings should be ignored in tests (@jsamoocha, #319)
+
+## v1.2.0
+
+### Added
```

Comment:
oh wait - i'm confusing myself. i shouldn't work when i'm tired. the other changelog in the pydantic branch has those commits. this looks ok to me. but i'd love a second look as i'm obviously not processing information correctly today 😆 i just want to ensure all of these items are associated with what is on master which they should be.

Reply:
Whenever we merged a PR on `master`, I rebased and force-pushed `pydantic_model`. This ensures the changelog in the latter branch is complete whenever we merge that into `master`.

I don't think there's anything to worry about re. the changelog (but I understand the confusion :-), I guess that's part of long-lasting feature branches (which I'm not a fan of, but was kinda inevitable in this case))

Reply:
ok perfect! i just had a moment but then i recognized i was tired last night.!!